### PR TITLE
switched bound funcs to arrow funcs for EdgeGroup.js

### PR DESCRIPTION
**Pull request by kenneth-miura**

### CODE REVIEW

File: js/components/graph/EdgeGroup.js [hunk 1]
```diff
@@ -9,16 +9,12 @@ export default class EdgeGroup extends React.Component {
...
-    this.reset = this.reset.bind(this);
-    this.generateEdge = this.generateEdge.bind(this);
-    this.setRefEntry = this.setRefEntry.bind(this);
   }
 
   // When an edge's state changes and the edge is not undefined,
   // it will call updateEdgeStatus and update EdgeGroup's state with its
   // edgeID and status. This function is passed as a props to Edge.
-  updateEdgeStatus(edgeID, state) {
+  updateEdgeStatus = (edgeID, state) =>{
```
Comment (david-yz-liu) [inline js/components/graph/EdgeGroup.js:17]: Please put a space before `{`

File: js/components/graph/EdgeGroup.js [hunk 2]
```diff
@@ -29,18 +25,18 @@ export default class EdgeGroup extends React.Component {
     });
   }
 
-  reset() {
+  reset = ()=> {
```
Comment (david-yz-liu) [inline js/components/graph/EdgeGroup.js:28]: Use a space after `()`

File: js/components/graph/EdgeGroup.js [hunk 3]
```diff
@@ -29,18 +25,18 @@ export default class EdgeGroup extends React.Component {
...
     this.props.edgesJSON.forEach(edgeJSON => {
       this[edgeJSON.id_].setState({ status: "inactive" });
     });
   }
 
-  setRefEntry(edgeJSON) {
+  setRefEntry = (edgeJSON) =>{
```
Comment (david-yz-liu) [inline js/components/graph/EdgeGroup.js:34]: Don't include parentheses around a single parameter; that is, you can write `edgeJSON => { ...`

**Review by david-yz-liu — COMMENTED** [review]
@kenneth-miura good work. I left a few style comments below!

File: js/components/graph/EdgeGroup.js (hunk 3 above)
Comment (kenneth-miura) [reply js/components/graph/EdgeGroup.js:34]: Thank you! Are some of these issues supposed to be automatically fixed by me running 'yarn run prettier'?

Comment (david-yz-liu) [reply js/components/graph/EdgeGroup.js:34]: Hmm not exactly, although I think the pre-commit hook is supposed to detect style issues like this one. For now you can fix these manually, and I'll check out the pre-commit configuration.

**Pull request merged**